### ISSUE TITLE
undefined method `merge' for "search":String

### ISSUE THREAD

**Issue by brodyhoskins** [activerecord-hackery/ransack#1269]
I'm hesitant to file an issue here but I can't seem to get to the bottom of this.

My environment is:
Rails 7.0.0
ransack (2.5.0)
ruby 3.0.3p157 (2021-11-24 revision 3fb7d2cadc) [arm64-darwin21]

I have a controller that searches various models on the same page (see Controller B below).

In my working controllers I have:

*Controller A (Working):*
```ruby
    @q = current_user
         .broker
         .broker_carriers
         .includes(
           [
             :broker_customer, { broker_customer: :customer },
             :carrier
           ]
         ).ransack(params[:q])

    @q.sorts = "#{Carrier.table_name.singularize}_name" if @q.sorts.blank?
    @pagy, @broker_carriers = pagy(@q.result)
```

The non-working controller contains:

*Controller B (Non-Working):*

```ruby
    @broker_carriers_q = current_user.broker.broker_carriers.ransack(carrier_name_cont: params[:q])
    @broker_carriers_q.sorts = 'carrier_name asc' if @broker_carriers_q.sorts.blank?
    @broker_carriers = @broker_carriers_q.result

    @broker_customers_q = current_user.broker.broker_customers.ransack(customer_name_cont: params[:q])
    @broker_customers_q.sorts = 'customer_name asc' if @broker_customers_q.sorts.blank?
    @broker_customers = @broker_customers_q.result

    @shipments_q = current_user.broker.shipments.ransack(
      bol_notes_or_broker_carrier_carrier_name_or_broker_carrier_carrier_scac_or_broker_customer_customer_name_or_broker_notes_or_load_id_or_shipper_name_or_shipper_contact_name_or_receiver_name_or_receiver_contact_name_or_shipper_address_city_or_receiver_address_city_or_shipper_address_zip_code_or_receiver_address_zip_code_or_pro_or_tracking_carrier_pro_cont: params[:q]
    )
    @shipments = @shipments_q.result
    @shipments_q.sorts = 'created_at desc' if @shipments_q.sorts.blank?
```

My views work by passing along the `Ransack::Search#result` result (to be paginated once I have this working) as well as the `q`:

```ruby
  <% unless @broker_carriers.blank? %>
    <h3>Carriers</h3>
    <%= render 'broker/broker_carriers/show_many', broker_carriers: @broker_carriers, q: @broker_carriers_q %>
  <% end %>
```

```ruby
            <th><%= sort_link q, :carrier_name, 'Carrier' %></th>
```

But I'm running into:

```
undefined method `merge' for "[REDACTED]":String

ransack (2.5.0) lib/ransack/helpers/form_helper.rb:174:in `search_and_sort_params' 
ransack (2.5.0) lib/ransack/helpers/form_helper.rb:135:in `url_options' 
ransack (2.5.0) lib/ransack/helpers/form_helper.rb:55:in `sort_link' 
app/views/broker/broker_carriers/_show_many.html.erb:12 
actionview (7.0.0) lib/action_view/helpers/capture_helper.rb:45:in `block in capture' 
actionview (7.0.0) lib/action_view/helpers/capture_helper.rb:209:in `with_output_buffer' 
actionview (7.0.0) lib/action_view/helpers/capture_helper.rb:45:in `capture' 
actionview (7.0.0) lib/action_view/helpers/tag_helper.rb:69:in `tag_string' 
actionview (7.0.0) lib/action_view/helpers/tag_helper.rb:157:in `method_missing' 
turbo-rails (1.0.0) app/helpers/turbo/frames_helper.rb:30:in `turbo_frame_tag' 
app/views/broker/broker_carriers/_show_many.html.erb:1 
actionview (7.0.0) lib/action_view/base.rb:244:in `public_send' 
actionview (7.0.0) lib/action_view/base.rb:244:in `_run' 
actionview (7.0.0) lib/action_view/template.rb:157:in `block in render' 
activesupport (7.0.0) lib/active_support/notifications.rb:206:in `block in instrument' 
activesupport (7.0.0) lib/active_support/notifications/instrumenter.rb:24:in `instrument' 
activesupport (7.0.0) lib/active_support/notifications.rb:206:in `instrument' 
actionview (7.0.0) lib/action_view/template.rb:361:in `instrument_render_template' 
actionview (7.0.0) lib/action_view/template.rb:155:in `render' 
actionview (7.0.0) lib/action_view/renderer/partial_renderer.rb:251:in `block in render_partial_template' 
activesupport (7.0.0) lib/active_support/notifications.rb:206:in `block in instrument' 
activesupport (7.0.0) lib/active_support/notifications/instrumenter.rb:24:in `instrument' 
activesupport (7.0.0) lib/active_support/notifications.rb:206:in `instrument' 
actionview (7.0.0) lib/action_view/renderer/partial_renderer.rb:246:in `render_partial_template' 
actionview (7.0.0) lib/action_view/renderer/partial_renderer.rb:237:in `render' 
actionview (7.0.0) lib/action_view/renderer/renderer.rb:81:in `render_partial_to_object' 
actionview (7.0.0) lib/action_view/renderer/renderer.rb:53:in `render_partial' 
actionview (7.0.0) lib/action_view/helpers/rendering_helper.rb:45:in `render' 
app/views/broker/search/search.html.erb:12 
actionview (7.0.0) lib/action_view/base.rb:244:in `public_send' 
actionview (7.0.0) lib/action_view/base.rb:244:in `_run' 
actionview (7.0.0) lib/action_view/template.rb:157:in `block in render' 
activesupport (7.0.0) lib/active_support/notifications.rb:206:in `block in instrument' 
activesupport (7.0.0) lib/active_support/notifications/instrumenter.rb:24:in `instrument' 
activesupport (7.0.0) lib/active_support/notifications.rb:206:in `instrument' 
actionview (7.0.0) lib/action_view/template.rb:361:in `instrument_render_template' 
actionview (7.0.0) lib/action_view/template.rb:155:in `render' 
actionview (7.0.0) lib/action_view/renderer/template_renderer.rb:65:in `block (2 levels) in render_template' 
activesupport (7.0.0) lib/active_support/notifications.rb:206:in `block in instrument' 
activesupport (7.0.0) lib/active_support/notifications/instrumenter.rb:24:in `instrument' 
activesupport (7.0.0) lib/active_support/notifications.rb:206:in `instrument' 
actionview (7.0.0) lib/action_view/renderer/template_renderer.rb:60:in `block in render_template' 
actionview (7.0.0) lib/action_view/renderer/template_renderer.rb:75:in `block in render_with_layout' 
activesupport (7.0.0) lib/active_support/notifications.rb:206:in `block in instrument' 
activesupport (7.0.0) lib/active_support/notifications/instrumenter.rb:24:in `instrument' 
activesupport (7.0.0) lib/active_support/notifications.rb:206:in `instrument' 
actionview (7.0.0) lib/action_view/renderer/template_renderer.rb:74:in `render_with_layout' 
actionview (7.0.0) lib/action_view/renderer/template_renderer.rb:59:in `render_template' 
actionview (7.0.0) lib/action_view/renderer/template_renderer.rb:11:in `render' 
actionview (7.0.0) lib/action_view/renderer/renderer.rb:61:in `render_template_to_object' 
actionview (7.0.0) lib/action_view/renderer/renderer.rb:29:in `render_to_object' 
actionview (7.0.0) lib/action_view/rendering.rb:117:in `block in _render_template' 
actionview (7.0.0) lib/action_view/base.rb:270:in `in_rendering_context' 
actionview (7.0.0) lib/action_view/rendering.rb:116:in `_render_template' 
actionpack (7.0.0) lib/action_controller/metal/streaming.rb:216:in `_render_template' 
actionview (7.0.0) lib/action_view/rendering.rb:103:in `render_to_body' 
actionpack (7.0.0) lib/action_controller/metal/rendering.rb:46:in `render_to_body' 
actionpack (7.0.0) lib/action_controller/metal/renderers.rb:142:in `render_to_body' 
actionpack (7.0.0) lib/abstract_controller/rendering.rb:25:in `render' 
actionpack (7.0.0) lib/action_controller/metal/rendering.rb:30:in `render' 
actionpack (7.0.0) lib/action_controller/metal/instrumentation.rb:22:in `block (2 levels) in render' 
/Users/brody/.rbenv/versions/3.0.3/lib/ruby/3.0.0/benchmark.rb:308:in `realtime' 
activesupport (7.0.0) lib/active_support/core_ext/benchmark.rb:14:in `ms' 
actionpack (7.0.0) lib/action_controller/metal/instrumentation.rb:22:in `block in render' 
actionpack (7.0.0) lib/action_controller/metal/instrumentation.rb:91:in `cleanup_view_runtime' 
activerecord (7.0.0) lib/active_record/railties/controller_runtime.rb:34:in `cleanup_view_runtime' 
actionpack (7.0.0) lib/action_controller/metal/instrumentation.rb:21:in `render' 
wicked_pdf (2.1.0) lib/wicked_pdf/pdf_helper.rb:46:in `call' 
wicked_pdf (2.1.0) lib/wicked_pdf/pdf_helper.rb:46:in `render_with_wicked_pdf' 
wicked_pdf (2.1.0) lib/wicked_pdf/pdf_helper.rb:30:in `render' 
actionpack (7.0.0) lib/action_controller/metal/implicit_render.rb:35:in `default_render' 
actionpack (7.0.0) lib/action_controller/metal/basic_implicit_render.rb:6:in `block in send_action' 
<internal:kernel>:90:in `tap' 
actionpack (7.0.0) lib/action_controller/metal/basic_implicit_render.rb:6:in `send_action' 
actionpack (7.0.0) lib/abstract_controller/base.rb:214:in `process_action' 
actionpack (7.0.0) lib/action_controller/metal/rendering.rb:53:in `process_action' 
actionpack (7.0.0) lib/abstract_controller/callbacks.rb:234:in `block in process_action' 
activesupport (7.0.0) lib/active_support/callbacks.rb:118:in `block in run_callbacks' 
actiontext (7.0.0) lib/action_text/rendering.rb:20:in `with_renderer' 
actiontext (7.0.0) lib/action_text/engine.rb:69:in `block (4 levels) in <class:Engine>' 
activesupport (7.0.0) lib/active_support/callbacks.rb:127:in `instance_exec' 
activesupport (7.0.0) lib/active_support/callbacks.rb:127:in `block in run_callbacks' 
activesupport (7.0.0) lib/active_support/callbacks.rb:138:in `run_callbacks' 
actionpack (7.0.0) lib/abstract_controller/callbacks.rb:233:in `process_action' 
actionpack (7.0.0) lib/action_controller/metal/rescue.rb:22:in `process_action' 
actionpack (7.0.0) lib/action_controller/metal/instrumentation.rb:67:in `block in process_action' 
activesupport (7.0.0) lib/active_support/notifications.rb:206:in `block in instrument' 
activesupport (7.0.0) lib/active_support/notifications/instrumenter.rb:24:in `instrument' 
activesupport (7.0.0) lib/active_support/notifications.rb:206:in `instrument' 
actionpack (7.0.0) lib/action_controller/metal/instrumentation.rb:66:in `process_action' 
actionpack (7.0.0) lib/action_controller/metal/params_wrapper.rb:259:in `process_action' 
activerecord (7.0.0) lib/active_record/railties/controller_runtime.rb:27:in `process_action' 
actionpack (7.0.0) lib/abstract_controller/base.rb:151:in `process' 
actionview (7.0.0) lib/action_view/rendering.rb:39:in `process' 
actionpack (7.0.0) lib/action_controller/metal.rb:188:in `dispatch' 
actionpack (7.0.0) lib/action_controller/metal.rb:251:in `dispatch' 
actionpack (7.0.0) lib/action_dispatch/routing/route_set.rb:49:in `dispatch' 
actionpack (7.0.0) lib/action_dispatch/routing/route_set.rb:32:in `serve' 
actionpack (7.0.0) lib/action_dispatch/journey/router.rb:50:in `block in serve' 
actionpack (7.0.0) lib/action_dispatch/journey/router.rb:32:in `each' 
actionpack (7.0.0) lib/action_dispatch/journey/router.rb:32:in `serve' 
actionpack (7.0.0) lib/action_dispatch/routing/route_set.rb:850:in `call' 
bullet (7.0.0) lib/bullet/rack.rb:15:in `call' 
warden (1.2.9) lib/warden/manager.rb:36:in `block in call' 
warden (1.2.9) lib/warden/manager.rb:34:in `catch' 
warden (1.2.9) lib/warden/manager.rb:34:in `call' 
rack (2.2.3) lib/rack/tempfile_reaper.rb:15:in `call' 
rack (2.2.3) lib/rack/etag.rb:27:in `call' 
rack (2.2.3) lib/rack/conditional_get.rb:27:in `call' 
rack (2.2.3) lib/rack/head.rb:12:in `call' 
actionpack (7.0.0) lib/action_dispatch/http/permissions_policy.rb:22:in `call' 
actionpack (7.0.0) lib/action_dispatch/http/content_security_policy.rb:18:in `call' 
rack (2.2.3) lib/rack/session/abstract/id.rb:266:in `context' 
rack (2.2.3) lib/rack/session/abstract/id.rb:260:in `call' 
actionpack (7.0.0) lib/action_dispatch/middleware/cookies.rb:693:in `call' 
activerecord (7.0.0) lib/active_record/migration.rb:603:in `call' 
actionpack (7.0.0) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call' 
activesupport (7.0.0) lib/active_support/callbacks.rb:99:in `run_callbacks' 
actionpack (7.0.0) lib/action_dispatch/middleware/callbacks.rb:26:in `call' 
actionpack (7.0.0) lib/action_dispatch/middleware/executor.rb:14:in `call' 
actionpack (7.0.0) lib/action_dispatch/middleware/actionable_exceptions.rb:17:in `call' 
rollbar (3.3.0) lib/rollbar/middleware/rails/rollbar.rb:25:in `block in call' 
rollbar (3.3.0) lib/rollbar.rb:145:in `scoped' 
rollbar (3.3.0) lib/rollbar/middleware/rails/rollbar.rb:22:in `call' 
actionpack (7.0.0) lib/action_dispatch/middleware/debug_exceptions.rb:28:in `call' 
rollbar (3.3.0) lib/rollbar/middleware/rails/show_exceptions.rb:22:in `call_with_rollbar' 
web-console (4.2.0) lib/web_console/middleware.rb:132:in `call_app' 
web-console (4.2.0) lib/web_console/middleware.rb:28:in `block in call' 
web-console (4.2.0) lib/web_console/middleware.rb:17:in `catch' 
web-console (4.2.0) lib/web_console/middleware.rb:17:in `call' 
actionpack (7.0.0) lib/action_dispatch/middleware/show_exceptions.rb:26:in `call' 
railties (7.0.0) lib/rails/rack/logger.rb:36:in `call_app' 
railties (7.0.0) lib/rails/rack/logger.rb:25:in `block in call' 
activesupport (7.0.0) lib/active_support/tagged_logging.rb:99:in `block in tagged' 
activesupport (7.0.0) lib/active_support/tagged_logging.rb:37:in `tagged' 
activesupport (7.0.0) lib/active_support/tagged_logging.rb:99:in `tagged' 
railties (7.0.0) lib/rails/rack/logger.rb:25:in `call' 
actionpack (7.0.0) lib/action_dispatch/middleware/remote_ip.rb:93:in `call' 
actionpack (7.0.0) lib/action_dispatch/middleware/request_id.rb:26:in `call' 
rack (2.2.3) lib/rack/method_override.rb:24:in `call' 
rack (2.2.3) lib/rack/runtime.rb:22:in `call' 
activesupport (7.0.0) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call' 
actionpack (7.0.0) lib/action_dispatch/middleware/server_timing.rb:20:in `call' 
actionpack (7.0.0) lib/action_dispatch/middleware/executor.rb:14:in `call' 
actionpack (7.0.0) lib/action_dispatch/middleware/static.rb:23:in `call' 
rack (2.2.3) lib/rack/sendfile.rb:110:in `call' 
actionpack (7.0.0) lib/action_dispatch/middleware/host_authorization.rb:137:in `call' 
webpacker (6.0.0.rc.6) lib/webpacker/dev_server_proxy.rb:25:in `perform_request' 
rack-proxy (0.7.0) lib/rack/proxy.rb:63:in `call' 
railties (7.0.0) lib/rails/engine.rb:530:in `call' 
puma (5.5.2) lib/puma/configuration.rb:249:in `call' 
puma (5.5.2) lib/puma/request.rb:77:in `block in handle_request' 
puma (5.5.2) lib/puma/thread_pool.rb:340:in `with_force_shutdown' 
puma (5.5.2) lib/puma/request.rb:76:in `handle_request' 
puma (5.5.2) lib/puma/server.rb:447:in `process_client' 
puma (5.5.2) lib/puma/thread_pool.rb:147:in `block in spawn_thread' 
```

I've been digging through `lib/ransack/helpers/form_helper.rb` quite a bit to debug the difference between controller A and controller B with some good old-fashioned `puts` statements. Here are those results:

*Controller A (Working):*
```console
-----------------> search_object: #<Ransack::Search:0x00000001092dc328>
------------------> search_params: {"sorts"=>"#<Ransack::Nodes::Sort:0x000000012dbb3130>", "carrier_name_cont"=>"[REDACTED STRING]"}
------------------> sort_params: shipments_count asc
```

*Controller B (Non-Working):*
```console
-----------------> search_object: #<Ransack::Search:0x000000010900fac8>
------------------> search_params: [REDACTED STRING]
------------------> sort_params: carrier_name desc
```

I've confirmed the view does indeed receive the right type of object by removing the `sort_link` and replacing it with:
```ruby
<%= q %>
```

It outputs:
```
#<Ransack::Search:0x00000001042262a8> Ransack::Search
```

I'm not entirely sure why passing a `Ransack::Search` object in one place works as intended but in another, it doesn't. I'm using the same call to `sort_link` since it's called by the same partial each time.

**Comment by deivid-rodriguez** [comment]
Would you be able to provide a reproducible example, so it can be investigated?

**Comment by deivid-rodriguez** [comment]
@brodyhoskins I think the problem is that your second controller is misusing the "q" parameter. How are you building the form where the search input is typed?

**Comment by deivid-rodriguez** [comment]
Closing due to lack of feedback.

**Comment by aiandrox** [comment]
@deivid-rodriguez 

I also got the same error.

My environment is:
Rails 6.1.4.6
ransack 3.1.0
ruby 3.0.2p107

Here's how to reproduce it.

controller:

```rb
class ProductsController < ApplicationController
  def index
    @q = Product.ransack(params[:q])
    @products = @q.result
  end
end
```

view:

```rb
<% if @products.present? %>
  <%= sort_link(@q, :id) %>
<% end %>
```

and access: `http://127.0.0.1:3000/products?q=info`

Normally people do not access that URL, but typing the URL directly will result in an error.

```
NoMethodError in Products#index
Showing /Users/k_end/private/development/fledge-hub/app/views/products/index.html.erb where line #2 raised:

undefined method `merge' for "info":String

ransack (3.1.0) lib/ransack/helpers/form_helper.rb:179:in `search_and_sort_params'
ransack (3.1.0) lib/ransack/helpers/form_helper.rb:134:in `url_options'
ransack (3.1.0) lib/ransack/helpers/form_helper.rb:54:in `sort_link'
app/views/products/index.html.erb:2
actionview (6.1.4.6) lib/action_view/base.rb:247:in `public_send'
actionview (6.1.4.6) lib/action_view/base.rb:247:in `_run'
actionview (6.1.4.6) lib/action_view/template.rb:154:in `block in render'
activesupport (6.1.4.6) lib/active_support/notifications.rb:205:in `instrument'
actionview (6.1.4.6) lib/action_view/template.rb:345:in `instrument_render_template'
actionview (6.1.4.6) lib/action_view/template.rb:152:in `render'
actionview (6.1.4.6) lib/action_view/renderer/template_renderer.rb:61:in `block (2 levels) in render_template'
activesupport (6.1.4.6) lib/active_support/notifications.rb:203:in `block in instrument'
activesupport (6.1.4.6) lib/active_support/notifications/instrumenter.rb:24:in `instrument'
activesupport (6.1.4.6) lib/active_support/notifications.rb:203:in `instrument'
actionview (6.1.4.6) lib/action_view/renderer/template_renderer.rb:56:in `block in render_template'
actionview (6.1.4.6) lib/action_view/renderer/template_renderer.rb:71:in `block in render_with_layout'
activesupport (6.1.4.6) lib/active_support/notifications.rb:203:in `block in instrument'
activesupport (6.1.4.6) lib/active_support/notifications/instrumenter.rb:24:in `instrument'
activesupport (6.1.4.6) lib/active_support/notifications.rb:203:in `instrument'
actionview (6.1.4.6) lib/action_view/renderer/template_renderer.rb:70:in `render_with_layout'
actionview (6.1.4.6) lib/action_view/renderer/template_renderer.rb:55:in `render_template'
actionview (6.1.4.6) lib/action_view/renderer/template_renderer.rb:11:in `render'
actionview (6.1.4.6) lib/action_view/renderer/renderer.rb:61:in `render_template_to_object'
actionview (6.1.4.6) lib/action_view/renderer/renderer.rb:29:in `render_to_object'
actionview (6.1.4.6) lib/action_view/rendering.rb:117:in `block in _render_template'
actionview (6.1.4.6) lib/action_view/base.rb:273:in `in_rendering_context'
actionview (6.1.4.6) lib/action_view/rendering.rb:116:in `_render_template'
actionpack (6.1.4.6) lib/action_controller/metal/streaming.rb:218:in `_render_template'
actionview (6.1.4.6) lib/action_view/rendering.rb:103:in `render_to_body'
actionpack (6.1.4.6) lib/action_controller/metal/rendering.rb:52:in `render_to_body'
actionpack (6.1.4.6) lib/action_controller/metal/renderers.rb:142:in `render_to_body'
actionpack (6.1.4.6) lib/abstract_controller/rendering.rb:25:in `render'
actionpack (6.1.4.6) lib/action_controller/metal/rendering.rb:36:in `render'
actionpack (6.1.4.6) lib/action_controller/metal/instrumentation.rb:46:in `block (2 levels) in render'
/Users/k_end/.anyenv/envs/rbenv/versions/3.0.2/lib/ruby/3.0.0/benchmark.rb:308:in `realtime'
activesupport (6.1.4.6) lib/active_support/core_ext/benchmark.rb:14:in `ms'
actionpack (6.1.4.6) lib/action_controller/metal/instrumentation.rb:46:in `block in render'
actionpack (6.1.4.6) lib/action_controller/metal/instrumentation.rb:86:in `cleanup_view_runtime'
activerecord (6.1.4.6) lib/active_record/railties/controller_runtime.rb:34:in `cleanup_view_runtime'
actionpack (6.1.4.6) lib/action_controller/metal/instrumentation.rb:45:in `render'
meta-tags (2.16.0) lib/meta_tags/controller_helper.rb:22:in `render'
actionpack (6.1.4.6) lib/action_controller/metal/implicit_render.rb:35:in `default_render'
actionpack (6.1.4.6) lib/action_controller/metal/basic_implicit_render.rb:6:in `block in send_action'
<internal:kernel>:90:in `tap'
actionpack (6.1.4.6) lib/action_controller/metal/basic_implicit_render.rb:6:in `send_action'
actionpack (6.1.4.6) lib/abstract_controller/base.rb:228:in `process_action'
actionpack (6.1.4.6) lib/action_controller/metal/rendering.rb:30:in `process_action'
actionpack (6.1.4.6) lib/abstract_controller/callbacks.rb:42:in `block in process_action'
activesupport (6.1.4.6) lib/active_support/callbacks.rb:117:in `block in run_callbacks'
active_decorator (1.4.0) lib/active_decorator/view_context.rb:37:in `block (3 levels) in <module:Filter>'
active_decorator (1.4.0) lib/active_decorator/view_context.rb:25:in `run_with'
active_decorator (1.4.0) lib/active_decorator/view_context.rb:36:in `block (2 levels) in <module:Filter>'
activesupport (6.1.4.6) lib/active_support/callbacks.rb:126:in `instance_exec'
activesupport (6.1.4.6) lib/active_support/callbacks.rb:126:in `block in run_callbacks'
actiontext (6.1.4.6) lib/action_text/rendering.rb:20:in `with_renderer'
actiontext (6.1.4.6) lib/action_text/engine.rb:59:in `block (4 levels) in <class:Engine>'
activesupport (6.1.4.6) lib/active_support/callbacks.rb:126:in `instance_exec'
activesupport (6.1.4.6) lib/active_support/callbacks.rb:126:in `block in run_callbacks'
activesupport (6.1.4.6) lib/active_support/callbacks.rb:137:in `run_callbacks'
actionpack (6.1.4.6) lib/abstract_controller/callbacks.rb:41:in `process_action'
actionpack (6.1.4.6) lib/action_controller/metal/rescue.rb:22:in `process_action'
actionpack (6.1.4.6) lib/action_controller/metal/instrumentation.rb:34:in `block in process_action'
activesupport (6.1.4.6) lib/active_support/notifications.rb:203:in `block in instrument'
activesupport (6.1.4.6) lib/active_support/notifications/instrumenter.rb:24:in `instrument'
activesupport (6.1.4.6) lib/active_support/notifications.rb:203:in `instrument'
actionpack (6.1.4.6) lib/action_controller/metal/instrumentation.rb:33:in `process_action'
actionpack (6.1.4.6) lib/action_controller/metal/params_wrapper.rb:249:in `process_action'
activerecord (6.1.4.6) lib/active_record/railties/controller_runtime.rb:27:in `process_action'
actionpack (6.1.4.6) lib/abstract_controller/base.rb:165:in `process'
actionview (6.1.4.6) lib/action_view/rendering.rb:39:in `process'
actionpack (6.1.4.6) lib/action_controller/metal.rb:190:in `dispatch'
actionpack (6.1.4.6) lib/action_controller/metal.rb:254:in `dispatch'
actionpack (6.1.4.6) lib/action_dispatch/routing/route_set.rb:50:in `dispatch'
actionpack (6.1.4.6) lib/action_dispatch/routing/route_set.rb:33:in `serve'
actionpack (6.1.4.6) lib/action_dispatch/journey/router.rb:50:in `block in serve'
actionpack (6.1.4.6) lib/action_dispatch/journey/router.rb:32:in `each'
actionpack (6.1.4.6) lib/action_dispatch/journey/router.rb:32:in `serve'
actionpack (6.1.4.6) lib/action_dispatch/routing/route_set.rb:842:in `call'
actionpack (6.1.4.6) lib/action_dispatch/middleware/static.rb:24:in `call'
bullet (7.0.1) lib/bullet/rack.rb:15:in `call'
exception_notification (11c1e919ea50) lib/exception_notification/rack.rb:49:in `call'
rack (2.2.3) lib/rack/tempfile_reaper.rb:15:in `call'
rack (2.2.3) lib/rack/etag.rb:27:in `call'
rack (2.2.3) lib/rack/conditional_get.rb:27:in `call'
rack (2.2.3) lib/rack/head.rb:12:in `call'
actionpack (6.1.4.6) lib/action_dispatch/http/permissions_policy.rb:22:in `call'
actionpack (6.1.4.6) lib/action_dispatch/http/content_security_policy.rb:18:in `call'
rack (2.2.3) lib/rack/session/abstract/id.rb:266:in `context'
rack (2.2.3) lib/rack/session/abstract/id.rb:260:in `call'
actionpack (6.1.4.6) lib/action_dispatch/middleware/cookies.rb:689:in `call'
activerecord (6.1.4.6) lib/active_record/migration.rb:601:in `call'
actionpack (6.1.4.6) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.4.6) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.4.6) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.4.6) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.6) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.4.6) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
web-console (4.2.0) lib/web_console/middleware.rb:132:in `call_app'
web-console (4.2.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (4.2.0) lib/web_console/middleware.rb:17:in `catch'
web-console (4.2.0) lib/web_console/middleware.rb:17:in `call'
actionpack (6.1.4.6) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.4.6) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.4.6) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.4.6) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.4.6) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.4.6) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.4.6) lib/rails/rack/logger.rb:26:in `call'
sprockets-rails (3.4.2) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (6.1.4.6) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.4.6) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/method_override.rb:24:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.4.6) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.4.6) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.6) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.4.6) lib/action_dispatch/middleware/host_authorization.rb:119:in `call'
rack-mini-profiler (2.3.3) lib/mini_profiler/profiler.rb:393:in `call'
webpacker (5.4.3) lib/webpacker/dev_server_proxy.rb:25:in `perform_request'
rack-proxy (0.7.2) lib/rack/proxy.rb:67:in `call'
railties (6.1.4.6) lib/rails/engine.rb:539:in `call'
puma (5.6.2) lib/puma/configuration.rb:252:in `call'
puma (5.6.2) lib/puma/request.rb:77:in `block in handle_request'
puma (5.6.2) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.6.2) lib/puma/request.rb:76:in `handle_request'
puma (5.6.2) lib/puma/server.rb:441:in `process_client'
puma (5.6.2) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
```

**Comment by deivid-rodriguez** [comment]
This one was fixed by #1374 and I plan to release it very soon as Ransack 4.0.
